.github/workflows/nuget.yaml

```diff
@@ -39,4 +39,11 @@ jobs:
       run: dotnet pack -c Release src/nuget.proj -o pkg --include-symbols
 
     - name: dotnet nuget push
-      run: dotnet nuget push pkg\*.nupkg -s https://www.nuget.org/ -k ${{ secrets.nuget_api_key }}
+      run: |
+        dotnet nuget push pkg\*.nupkg -s https://nuget.pkg.github.com/$env:GITHUB_REPOSITORY_OWNER -k ${{ secrets.GITHUB_TOKEN }}
+        dotnet nuget push pkg\*.nupkg -s https://www.nuget.org/ -k ${{ secrets.nuget_api_key }}
+        
+    - name: Delete Package Versions
+      uses: actions/delete-package-versions@v2
+      with:
+        min-versions-to-keep: 10
```

Comment:
Is this a real delete or unlist likes nuget.org?

Reply:
This will permanently delete the package, hmm, seems not friendly anyway.
still recommend users to download from nuget.org at moment

maybe make gh nuget for early access versions

Reply:
Get it, thanks